test/chrono-test.cc

```diff
@@ -543,15 +543,14 @@ TEST(chrono_test, negative_durations) {
 }
 
 TEST(chrono_test, special_durations) {
-  EXPECT_EQ(
-      "40.",
-      fmt::format("{:%S}", std::chrono::duration<double>(1e20)).substr(0, 3));
+  auto value = fmt::format("{:%S}", std::chrono::duration<double>(1e20));
+  // No decimal point is printed so size() is 2.
+  EXPECT_EQ(value.size(), 2);
+  EXPECT_EQ("40", value.substr(0, 2));
```

Comment:
Let's simplify this:

```c++
EXPECT_EQ(value, "40");
```
Then we don't need a comment because it's clear that we don't emit a decimal point.

Reply:
Of course. I recall the size of this string being platform dependent but after the changes in this PR it should always be consistent. Done